avaframe/com1DFA/com1DFA.py

```diff
@@ -111,147 +151,148 @@ def com1DFAMain(avalancheDir, cfgMain, cfgFile=''):
+    # keep for now for testing
+    inputSimFilesTest = inputSimFiles.copy()
 
-    # initialise reportDictList and flag indicating whether simulations have been performed
+    # initialize reportDict list
     reportDictList = []
-    simsPerformed = False
 
-    # loop over cfg files of all release scenarios
-    for cfgFileRel in cfgFilesRels:
+    # is there any simulation to run?
+    if bool(simDict):
 
-        log.debug('Full cfg file %s' % cfgFileRel)
+        # reset simDF and timing
+        simDF = ''
+        tCPUDF = ''
 
-        # copy inputSimFilesAll - as this becomes changed according to current release scenario
-        inputSimFiles = inputSimFilesAll.copy()
+        # loop over all simulations
+        for cuSim in simDict:
 
-        # reset variationDict
-        variationDict = ''
+            # load configuration object for current sim
+            cfg = simDict[cuSim]['cfgSim']
 
-        # get information on simulations that shall be performed according to parameter variation
-        modCfg, variationDict = dP.getParameterVariationInfo(avalancheDir, com1DFA, cfgFileRel)
+            # check configuraton for consistency
+            checkCfg.checkCfgConsistency(cfg)
 
-        # select release input data according to chosen release scenario
-        inputSimFiles = gI.selectReleaseScenario(inputSimFiles, modCfg['INPUT'])
+            # fetch simHash for current sim
+            simHash = simDict[cuSim]['simHash']
+            # append configuration to dataframe
+            simDF = cfgUtils.appendCgf2DF(simHash, cuSim, cfg, simDF)
 
-        # TODO: once it is confirmed that inputSimFiles is not changed within sim
-        # keep for now for testing
-        inputSimFilesTest = inputSimFiles.copy()
-        # first remove demFile entry as this is removed once the simulation DEMs are set
-        inputSimFilesTest.pop('demFile')
+            # log simulation name
+            log.info('Run simulation: %s' % cuSim)
 
-        # create a list of simulations and generate an individual configuration object for each simulation
-        # if need to reproduce exactly the hash - need to be strings with exactely the same number of digits!!
-        # first get info on already existing simulations in Outputs
-        simDFOld, simNameOld = cfgUtils.readAllConfigurationInfo(avalancheDir, specDir='')
+            # ++++++++++PERFORM com1DFA SIMULAITON++++++++++++++++
+            dem, reportDict, cfgFinal, tCPU, inputSimFilesNEW, particlesList, fieldsList, tSave = com1DFA.com1DFACore(cfg,
+                avalancheDir, cuSim, inputSimFiles, outDir, simHash=simHash)
+            simDF.at[simHash, 'nPart'] = str(int(particlesList[0]['nPart']))
 
-        # prepare simulations to run (only the new ones)
-        simDict = prepareVarSimDict(modCfg, inputSimFiles, variationDict, simNameOld=simNameOld)
+            # TODO check if inputSimFiles not changed within sim
```

Comment:
do we still need the todo?

Reply:
good question ;)

Reply:
currently the inputSimfiles ara checked in the end

Reply:
leave check in